src/uucore/src/lib/features/encoding.rs

```diff
@@ -42,138 +30,115 @@ pub enum Format {
     Base2Msbf,
     Z85,
 }
-use self::Format::*;
 
-const BASE2LSBF: Encoding = new_encoding! {
+pub const BASE2LSBF: Encoding = new_encoding! {
     symbols: "01",
     bit_order: LeastSignificantFirst,
 };
-const BASE2MSBF: Encoding = new_encoding! {
+
+pub const BASE2MSBF: Encoding = new_encoding! {
     symbols: "01",
     bit_order: MostSignificantFirst,
 };
 
-pub fn encode(f: Format, input: &[u8]) -> Result<String, EncodeError> {
-    Ok(match f {
-        Base32 => BASE32.encode(input),
-        Base64 => BASE64.encode(input),
-        Base64Url => BASE64URL.encode(input),
-        Base32Hex => BASE32HEX.encode(input),
-        Base16 => HEXUPPER.encode(input),
-        Base2Lsbf => BASE2LSBF.encode(input),
-        Base2Msbf => BASE2MSBF.encode(input),
-        Z85 => {
-            // According to the spec we should not accept inputs whose len is not a multiple of 4.
-            // However, the z85 crate implements a padded encoding and accepts such inputs. We have to manually check for them.
-            if input.len() % 4 == 0 {
-                z85::encode(input)
-            } else {
-                return Err(EncodeError::Z85InputLenNotMultipleOf4);
-            }
-        }
-    })
+pub struct ZEightFiveWrapper {}
```

Comment:
Is there a reason why this isn't just called `Z85Wrapper`, as one would expect for z85 encoding?

Reply:
Will change